woocommerce/woocommerce-android
# Issue/3624. Customer's list infra

Resolves #3624 

Infrastructure for the customer's list screen

**Testing**
1. Go to the orders screen
2. Click FAB
3. Click "START CUSTOMER CREATION"
4. Add customer screen should be visible


<img src="https://user-images.githubusercontent.com/4923871/110232041-481bc480-7f2c-11eb-8180-73b57983cfca.png" width="300"/>
<img src="https://user-images.githubusercontent.com/4923871/110232029-33d7c780-7f2c-11eb-86d3-4572b0c6a56b.png" width="300"/>
